### DISCUSSION TITLE
Can i vary or disable the data retention policy?

### DISCUSSION THREAD

**Discussion by alexandreoliveira** [globaleaks/globaleaks-whistleblowing-software#4558] [Q&A] [answered]
### What version of GlobaLeaks are you using?

 4.10.14

### What browser(s) are you seeing the problem on?

Chrome

### What operating system(s) are you seeing the problem on?

Windows, Linux

### Describe the issue

I have Expired submissions and after that desappear. Is necessary save the history of submissions, after expired is saved to history.
Is possible remove or disable this configuration?
If is not possible i need change the application, because the solution have protetion of subsmissions is globaleaks. 

### Proposed solution

Possible disable to remove expired submissions or possible disable date of expired.

**Comment by mauriziobulgarini** [comment]
If I understand correctly your question, you actually can disable the expiration date of your submissions: you just need to set to 0 the field "Expiration date" in channel settings.
![image](https://github.com/globaleaks/GlobaLeaks/assets/20863823/affbef65-14ab-4772-94f9-d2c93e053c1c)

**Comment by alexandreoliveira** [comment]
yes is this.
Many Thanks.

**Comment by evilaliv3** [comment, marked as answer]
Thank you @alexandreoliveira for the question and @mauriziobulgarini for the support you provided in the past.

I'm transforming this as a Q/A since i consider that many users time to time require this information.

It is possible to change the data retention policy for every context like shown below in the image shared by @mauriziobulgarini 
Disabling the data retention policy is a possibility but we highly recommend to do this on preproduction and for testing purposes. A live system for reasons of compliance (especially GDPR) would always need to feature a proper data retention policy.
